src/pint/utils.py

```diff
@@ -1580,21 +1620,246 @@ def companion_mass(pb, x, inc=60.0 * u.deg, mpsr=1.4 * u.solMass):
+        / (mp + mc) ** (4.0 / 3)
+    )
+    return value.to(u.s)
+
+
+@u.quantity_input
+def OMDOT(mp: u.Msun, mc: u.Msun, pb: u.d, e: u.dimensionless_unscaled):
+    """Post-Keplerian longitude of periastron precession rate OMDOT, assuming general relativity.
+
+    OMDOT (:math:`\dot \omega`) is the relativistic advance of periastron.
+
+    Parameters
+    ----------
+    mp : astropy.units.Quantity
+        pulsar mass
+    mc : astropy.units.Quantity
+        companion mass
+    pb : astropy.units.Quantity
+        Binary orbital period
+    e : astropy.units.Quantity or float
+        orbital eccentricity
+
+    Returns
+    -------
+    omdot : astropy.units.Quantity
+        In ``u.deg/u.yr``
+
+    Raises
+    ------
+    astropy.units.UnitsError
+        If the input data are not appropriate quantities
+    TypeError
+        If the input data are not quantities
+
+    Notes
+    -----
+    Calculates
+
+    .. math::
+
+        \dot \omega = 3T_{\odot}^{2/3} \\left(\\frac{P_b}{2\pi}\\right)^{-5/3}
+        \\frac{1}{1-e^2}(m_p+m_c)^{2/3}
+        
+    with :math:`T_\odot = GM_\odot c^{-3}`.
+
+    See [1]_
+
+    .. [1] Lorimer & Kramer, 2008, "The Handbook of Pulsar Astronomy", Eqn. 8.48
+
+    """
+    value = (
+        3
+        * (pb / (2 * np.pi)) ** (-5.0 / 3)
+        * (1 / (1 - e ** 2))
+        * (const.G * (mp + mc) / const.c ** 3) ** (2.0 / 3)
+    )
+    return value.to(u.deg / u.yr, equivalencies=u.dimensionless_angles())
+
+
+@u.quantity_input(pb=u.d, mp=u.Msun, mc=u.Msun, i=u.deg)
+def A1SINI(mp, mc, pb, i=90 * u.deg):
+    """Pulsar's semi-major axis.
+
+    The full semi-major axis is given by Kepler's third law.  This is the
+    projection (:math:`\sin i`) of just the pulsar's orbit (:math:`m_c/(m_p+m_c)`
+    times the full semi-major axis), which is what pulsar timing measures.
+
+    Parameters
+    ----------
+    mp : astropy.units.Quantity
+        pulsar mass
+    mc : astropy.units.Quantity
+        companion mass
+    pb : astropy.units.Quantity
+        Binary orbital period
+    i : astropy.units.Quantity
+        orbital inclination
+
+    Returns
+    -------
+    A1SINI : astropy.units.Quantity
+        Projected semi-major axis of pulsar's orbit in ``pint.ls``
+
+    Raises
+    ------
+    astropy.units.UnitsError
+        If the input data are not appropriate quantities
+    TypeError
+        If the input data are not quantities
+
+    Notes
+    -----
+    Calculates
+
+    .. math::
+
+        \\frac{a_p \sin i}{c} = \\frac{m_c \sin i}{(m_p+m_c)^{2/3}}
+        G^{1/3}\\left(\\frac{P_b}{2\pi}\\right)^{2/3}
+
+    See [1]_
+
+    .. [1] Lorimer & Kramer, 2008, "The Handbook of Pulsar Astronomy", Eqn. 8.21, 8.22, 8.27
+
+    """
+    return (
+        (mc * np.sin(i))
+        * (const.G * (pb / (2 * np.pi)) ** 2 / (mp + mc) ** 2) ** (1.0 / 3)
+    ).to(pint.ls)
+
+
+@u.quantity_input
+def ELL1_check(
+    A1: u.cm, E: u.dimensionless_unscaled, TRES: u.us, NTOA: int, outstring=True
```

Comment:
Lowercase would be preferable for arguments, for pep8 reasons, unless there's a strong reason to use uppercase. I could see an argument for A1 or E, but NTOA and TRES needn't be uppercase.

Reply:
I didn't write this function, just added decorators and improved the comments a bit (I think they need a reference or better explanation as well, but I didn't dig one up).  So I would be fine with lowercase but am wary about changing the API on other people's code.